.github/scripts/test_badge_consistency.py

```
@@ -66,17 +80,16 @@ def test_version_badge_drift_fails(self):
         self.assertTrue(any("version" in e.lower() for e in G.consistency_errors(**f)))
 
     def test_count_badge_drift_fails(self):
-        f = self._facts(); f["badge_counts"]["commands"] = 1
-        self.assertTrue(any("command" in e.lower() for e in G.consistency_errors(**f)))
+        f = self._facts(); f["badge_counts"]["core_lanes"] = 2
```

Ruff (0.16.3)

[error] Multiple statements on one line (semicolon)

(E702)